assets/addon_browser/addons.conf

```diff
@@ -758,4 +758,15 @@
       "github": "https://github.com/TechnicJelle/BlueMapWebAddons/tree/main/copy-coordinates"
     }
   }
+  {
+    name: "BluemapFrontiers"
+    description: "MapFrontiers integration for BlueMap"
+    author: "ErdbeerbaerLP"
+    apiVersion: ""
+    platforms: ["forge","fabric","quilt"]
```

Comment:
It looks like you're using neoforge, instead of normal forge. Maybe you can write that instead?
It's true that there is no other neoforge mod on the list, yet, so you may need to add an icon for it in here: https://github.com/BlueMap-Minecraft/BlueMapWiki/tree/master/assets/addon_browser/platforms

Reply:
I added the neoforge icon :)
So you only need to change the "forge" to "neoforge"

Reply:
I plan on supporting normal forge with older MC Versions soon. Will *add* neoforge tho, so I don't have to do another PR for that ;)